draft-ietf-mls-architecture.md

```diff
@@ -795,18 +795,24 @@ messages to the Delivery Service or the MLS clients.
 
 ### Message Suppression and Error Correction
 
-The MLS protocol is particularly sensitive about group operation
-message loss and reordering. This is because in the default setting,
-MLS clients have to process those specific messages in order to have a
-synchronized group state, after what the MLS protocol efficiently
-generates keys for application messages.
-[[TODO: It is unclear from this text whether MLS is "sensitive" in that it
-provides additional constraints to prevent this, or is "sensitive" in that it is
-vulnerable. Need to clarify]]
-
-The Delivery Service can have the role of helping with reliability,
-but is mainly useful for reliability in the asynchronous aspect of the
-communication between MLS clients.
+As noted above, MLS is designed to provide some robustness in the face of
+tampering within the secure transport, i.e., tampering by the Delivery Service.
+The confidentiality and authenticity properties of MLS prevent the DS reading or
```

Comment:
This is suggesting that the DS is a service running on a server outside of the client, where with #87, the definition would shift towards the DS being more of an abstract layer on top of an MLS client.